### PR TITLE
[#2788] fix(ci):  Fix CI issue after Github CI image is upgraded

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Change `docker-compose` to `docker compose`

### Why are the changes needed?

Github CI image updates its base image to upgrade docker compose from v1 to v2, in which the command `docker-compose` is no longer supported, instead user should use `docker compose`.

Fix: #2788

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

Test locally and CI.

### CODE REVIEW

File: .github/workflows/backend-integration-test.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ jobs:
           name: integrate test reports
           path: |
             build/reports
-            integration-test/build/integration-test.log
+            integration-test/build/integration-test-integration-test.log
```
Comment (jerryshao) [inline .github/workflows/backend-integration-test.yml:100]: Why do we need two "integration-test"?

Comment (yuqi1129) [reply .github/workflows/backend-integration-test.yml:100]: The log name is `param.systemProperty("gravitino.log.path", project.buildDir.path + "/${project.name}-integration-test.log")` for all modules that have integration test and the project name is `integrate-test`

Comment (jerryshao) [reply .github/workflows/backend-integration-test.yml:100]: I see.

**Review by mchades — APPROVED** [review]

**Comment by jerryshao** [comment]
We should have a reason for the failure. As @yuqi1129 discovered that docker-compose command also exists in CI environment.

**Comment by jerryshao** [comment]
@yuqi1129 @diqiu50 can you please another issue a) add some check mechanism into the CI script to check the availability of some necessary tools; b) improve the error message to better understand what's going on when CI is failed.

**Review by diqiu50 — APPROVED** [review]
LGTM

**Pull request merged**